perception/compare_map_segmentation/README.md

```diff
@@ -78,7 +78,7 @@ This filter is a combination of the distance_based_compare_map_filter and voxel_
 
 | Name                            | Type  | Description                                                                                                                             | Default value |
 | :------------------------------ | :---- | :-------------------------------------------------------------------------------------------------------------------------------------- | :------------ |
-| `use_dynamic_map_loading`       | bool  | map loading mode selection, `true` for dynamic map loading, `false` for static map loading                                              | true          |
+| `use_dynamic_map_loading`       | bool  | map loading mode selection, `true` for dynamic map loading, `false` for static map loading, recommended for no-split map pointcloud     | true          |
```

Comment:
Do you mean using no-split map pointcloud with `use_dynamic_map_loading:=true` is not harmful but it is meaning less and that it is not recommended?

Reply:
Yes it is not harmful to set` use_dynamic_map_loading:=true`. However then, compare_map_filter subscribes` /localization/initialization_state` and wait until ego-vehicle position is initialized to start loading map and voxelize map pointcloud. Set `use_dynamic_map_loading:=false` lets voxelization start right away after pointcloud_map loaded.

Reply:
Got it thank you!